packages/terra-site/webpack.prod.config.js

```diff
@@ -16,9 +17,13 @@ config.plugins.push(new webpack.DefinePlugin({
 // Clean build before running
 config.plugins.push(new CleanPlugin('build', { exclude: ['stats.json'] }));
 
+// Minify css and js
+config.plugins.push(new webpack.LoaderOptionsPlugin({ minimize: true }));
+
```

Comment:
I think we also want to add the UglifyJsPlugin.

https://webpack.js.org/guides/production-build/#simple-approach